CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ The following is a set of guidelines for contributing to BigCommerce's Developer
 
 The easiest way to edit a file is using GitHub's web interface:
 
-1. Navigate to the file in GitHub. For example, [making-requests.md](https://github.com/bigcommerce/dev-docs/blob/main/docs/api-docs/getting-started/making-requests.mdx).
+1. Navigate to the file in GitHub. For example, [making-requests.md](https://github.com/bigcommerce/docs/blob/main/CONTRIBUTING.md).
```

Comment:
```suggestion
1. Navigate to the file in GitHub. For example, [store-logs.mdx](https://github.com/bigcommerce/docs/blob/main/docs/api-docs/store-logs/store-logs.mdx).
```

Reply:
up to you on this I suppose - I merely used the content already there which was referencing the making-requests file, and repointed it to the cotributing file. (therefore the making changes was referencing the location that showed how to make changes etc?)

Reply:
The idea of this step is to provide an arbitrary file that one might choose to edit. The document that this step is written in is the same one that you linked to, so it wouldn't go anywhere.